contracts/test/WETH9.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import '../interfaces/external/IWETH.sol';
+
+contract WETH9 is IWETH {
+    string public name     = "Wrapped Ether";
+    string public symbol   = "WETH";
+    uint8  public decimals = 18;
+
+    mapping (address => uint)                       public  balance;
+    mapping (address => mapping (address => uint))  public  allowances;
```

Comment:
It's called allowance not allowances. If you rename it allowance, you don't need to implement the function allowance() since public variables get their public read function definitions by default

Reply:
ok